tests/resources/application.yml

```diff
@@ -124,6 +126,8 @@ logging:
 samples:
   home: ${instrument.calibration.home}/CalibrantSamples
 
+temp:
+  directory: ~/SNAPRed
```

Comment:
Will this overwrite the code folder?
Will this write the workspaces into the code folder? Should it instead point to the tests/resources/outputs folder?